### DISCUSSION TITLE
Epic Online Services

### DISCUSSION THREAD

**Discussion by kone9** [nxrighthere/UnrealCLR#180] [General]
Hi, I'm researching that unreal engine has an awesome tool called "Epic Online Services".
Do you think there are possibilities to use it with C # inside unreal engine?
What's your opinion about it?
Thank you.

https://www.youtube.com/watch?v=Og5TFxZOb1E

![Screenshot_10](https://user-images.githubusercontent.com/12847027/130337665-5fa4946a-4352-4043-abd9-9c26f8a269d2.jpg)
![Screenshot_11](https://user-images.githubusercontent.com/12847027/130337667-6143951d-6475-4764-b082-b47ef60e7474.jpg)
![Screenshot_13](https://user-images.githubusercontent.com/12847027/130337668-f6912165-3de2-48e6-b865-dccf5e66a7ac.jpg)

http://dev.epicgames.com/en-US/services-games

**Comment by nxrighthere** [comment]
> Do you think there are possibilities to use it with C # inside unreal engine?

[Online subsystem](https://docs.unrealengine.com/4.27/en-US/API/Plugins/OnlineSubsystem/) API is planned. Most likely, it will be exposed after finishing https://github.com/nxrighthere/UnrealCLR/issues/38.

> What's your opinion about it?

I have experience with this platform. It's too limited in terms of resources provided for a particular service. For a project aimed at a high number of concurrent users simultaneously accessing the game, there's a risk to hit usage limitations. You can find it in the [documentation](https://dev.epicgames.com/docs/services/en-US/GameServices/index.html) for every service at the bottom of the page.
